libs/prefill/near_full_data_storage.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python  # pylint: disable=C0302
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+"""
+disk near full data storage utility methods
+"""
+import logging
+
+from commons.helpers.health_helper import Health
+from commons.helpers.pods_helper import LogicalNode
+from config import CMN_CFG
+from config.s3 import S3_CFG
+from libs.durability.disk_failure_recovery_libs import DiskFailureRecoveryLib
+from scripts.s3_bench import s3bench
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DiskNearFullStorage:
```

Comment:
Remove the work Disk from class name as it misleads. NearFullStorage would suffice.